### PR TITLE
Aurora serverless data API for SQLAlchemy

### PR DESCRIPTION
**Pull request by revmischa**
Add aurora serverless DB that app can connect to using AWS API instead of psycopg2 low level connection. We can get rid of psycopg2 eventually for cloud deployments, though not local dev.

Add optional bastion EC2 instance for connecting to database via SSH (read more here https://aws.amazon.com/blogs/database/deploy-an-amazon-aurora-postgresql-db-cluster-with-recommended-best-practices-using-aws-cloudformation/)

Add test for db init / migrate / seed
Add initial migration

Remove lambda from VPC
Remove NAT GW from VPC (saves money, improves availability)

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -9,42 +9,52 @@ plugins:
...
+    sshKeyPairName: ''  # set this to enable bastion ec2 instance
+    alertEmails: 'me@mydomain.xyz'  # get SNS notifications of alarms at this address
+    banner: https://ops.jetbridge.com/public/banner.txt
+    bootstrap: https://ops.jetbridge.com/public/bastion_bootstrap.sh
+    instanceType: t3a.nano
+    tenancy: default
+    bannerEnabled: 'true'
+    tcpForwardingEnabled: 'true'
+    x11ForwardingEnabled: 'false'
+    logRetentionInDays: 731
```
Comment (janjur) [inline serverless.yml]: 2 years? A bit long and expensive IMO.

File: serverless.yml [hunk 2]
```diff
@@ -9,42 +9,52 @@ plugins:
...
-      - Ref: ServerlessSubnetA
-      - Ref: ServerlessSubnetB
-      - Ref: ServerlessSubnetC
+  network:
+    classB: 10 # second octet of VPC CIDR block (10.classB.0.0/16)
+  bastion:  # EC2 host to access database
+    sshKeyPairName: ''  # set this to enable bastion ec2 instance
+    alertEmails: 'me@mydomain.xyz'  # get SNS notifications of alarms at this address
+    banner: https://ops.jetbridge.com/public/banner.txt
+    bootstrap: https://ops.jetbridge.com/public/bastion_bootstrap.sh
```
Comment (janjur) [inline serverless.yml]: Have parts of this file been modified for this ticket or is it equal to amazon version? Should I get familiar with it as well?

Comment (revmischa) [reply serverless.yml]: There's nothing JB-specific in here no

File: serverless.yml (hunk 1 above)
Comment (revmischa) [reply serverless.yml]: I have no idea what it costs, but this should be very small

File: README.md [hunk 3]
```diff
@@ -41,32 +46,29 @@ poetry shell  # activate python virtual environment
 poetry install  # install dependencies
 ```
 
-### Do Stuff:
+### Run Dev Server:
```
[the remark this answers is not in the document]
Comment (revmischa) [reply README.md]: yeah no dev server on AWS

File: TEMPLATE/test/test_db_seed.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+from TEMPLATE.db.fixtures import seed_db
+from TEMPLATE.db import db
+
+
+def test_db_init_seed(app):
+    """Try initializing and seeding development database."""
+    drop_all_tables(app=app)
+    db.create_all(app=app)
+    seed_db()
+    drop_all_tables(app=app)
```
Comment (gnudeb) [inline TEMPLATE/test/test_db_seed.py]: Is the last `drop_all_tables(app=app)` a mistake? Looks wrong to me

Comment (revmischa) [reply TEMPLATE/test/test_db_seed.py]: it's testing that drop_all_tables() works

Comment (gnudeb) [reply TEMPLATE/test/test_db_seed.py]: Ooooh, I didn't notice that this a test 🤦‍♂

**Pull request merged**